appveyor.yml

```diff
@@ -0,0 +1,70 @@
+version: 0.0.0+{build}
+
+skip_tags: true
+
+skip_branch_with_pr: true
+
+build:
+  verbosity: minimal
+
+test: off
+
+environment:
+  global:
+    PUBLISH: False
+  matrix:
+  # TODO: Get builds working on PowerShell 7.2 on Windows.
+  # - job_name: PowerShell 7.2 on Windows
+  #   job_group: pwsh
+  #   appveyor_build_worker_image: Visual Studio 2022
+
+  # TODO: Get builds working on macOS.
+  # - job_name: PowerShell 7.1 on macOS
+  #   job_group: pwsh
+  #   appveyor_build_worker_image: macOS

```

Comment:
These changes broke the macOS build?

Reply:
I looked back two years in AppVeyor and didn't find any macOS builds. I was surprised.